examples/minimal/packages/client-react/src/index.tsx

```diff
@@ -2,18 +2,32 @@ import ReactDOM from "react-dom/client";
 import { App } from "./App";
 import { setup } from "./mud/setup";
 import { MUDProvider } from "./MUDContext";
-import { mount as mountDevTools } from "@latticexyz/dev-tools";
+import storeConfig from "contracts/mud.config";
 
 const rootElement = document.getElementById("react-root");
 if (!rootElement) throw new Error("React root not found");
 const root = ReactDOM.createRoot(rootElement);
 
 // TODO: figure out if we actually want this to be async or if we should render something else in the meantime
-setup().then((result) => {
+setup().then(async (result) => {
   root.render(
     <MUDProvider value={result}>
       <App />
     </MUDProvider>
   );
-  mountDevTools();
+
+  if (import.meta.env.DEV) {
+    const { mount: mountDevTools } = await import("@latticexyz/dev-tools");
+    mountDevTools({
+      config: storeConfig,
+      publicClient: result.network.publicClient,
+      walletClient: result.network.walletClient,
+      latestBlock$: result.network.latestBlock$,
+      blockStorageOperations$: result.network.blockStorageOperations$,
+      worldAddress: result.network.worldContract.address,
+      worldAbi: result.network.worldContract.abi,
+      write$: result.network.write$,
+      recsWorld: result.network.world,
+    });
```

Comment:
@alvrs thoughts on this?

Reply:
looks fine, maybe add a comment linking to https://vitejs.dev/guide/env-and-mode.html near `import.meta.env.DEV`?